### ISSUE TITLE
Alt text not being shown in Mastodon after updating post

### ISSUE THREAD

**Issue by quazar-omega** [pixelfed/pixelfed#4623]
I've updated a post of mine to add the alt text that was missing, but it doesn't seem like it is visible in Mastodon
